Problem2.rb

```diff
@@ -0,0 +1,31 @@
+# 2 largest numbers from integer array
+
+
+def TwoBiggestNumbers numbers
+
+
+
+   if numbers.nil?

```

Comment:
Why will you send the numbers nil? again the problem statement is clear about this, you should not need this check.